### PR TITLE
feature: add show_form model and migration

### PR DESCRIPTION
**Pull request by Neilxx**

### CODE REVIEW

File: backend/api/models/shown_form.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+        choices=bear_behavior_list, 
+        help_text='目擊當下熊在做什麼'
+    )
+    bear_behavior_text = models.CharField(
+        blank=True, 
+        null=True, 
+        max_length=255, 
+        help_text='目擊當下熊在做什麼-文字補充'
+    )
+    food = ArrayField(
```
Comment (Neilxx) [inline backend/api/models/shown_form.py:120]: 複選，這樣寫應該會 work？

File: backend/api/models/shown_form.py [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+        models.IntegerField(
+            blank=True, 
+            null=True, 
+            choices=food_list, 
+            help_text='黑熊在吃什麼'
+        )
+    )
+    food_object = JSONField()
```
Comment (Neilxx) [inline backend/api/models/shown_form.py:128]: 截圖 2022-07-07 下午3 04 41
current data from frontend as above.
not sure if this is the way to do it.
could be discussed 🤔

Comment (Neilxx) [reply backend/api/models/shown_form.py:128]: 截圖 2022-07-07 下午3 08 33

**Comment by Neilxx** [comment]
[最新版問卷_v20220304](https://github.com/OH-SHOWN/ohshown-backend/files/9061220/Final._20220304.pdf)

**Comment by Neilxx** [comment]
Currently lack of foreign key reference to main table, ohshown_event.
As ohshown_event model could be modify or replace by ohshown_event_next, the foreign key will be set after ohshown_event model finished.

File: backend/api/models/shown_form.py (hunk 1 above)
Comment (tai271828) [reply backend/api/models/shown_form.py:120]: Yes!

**Review by tai271828 — APPROVED** [review]
Looks good me. Replied some questions inline. I will merge if @EagleC0318 has no objection.

**Review by EagleC0318 — APPROVED** [review]
Looks good for me! Thanks for the contribution 😄

**Pull request merged**